README.md

```diff
@@ -129,6 +129,49 @@ This plugin activates a feature in the CMS that allows Mail template files to us
 * **mail-notify-ru.htm** will contain the mail template in Russian.
 * **mail-notify-fr.htm** will contain the mail template in French.
 
+## Extending a plugin with translatable fields
+If you are extending a plugin and want the added fields in the backend to be translatable, you have to use the '[backend.form.extendFieldsBefore](https://wintercms.com/docs/events/event/backend.form.extendFieldsBefore)' and tell which fields you want to be translatable by pushing them to the array.
+
+    public function boot() {
+        Event::listen('backend.form.extendFieldsBefore', function($widget) {
+            
+            // Only apply this listener when the Page controller is being used
+            if (!$widget->getController() instanceof \Winter\Pages\Controllers\Index) {
+                return;
+            }
+
+            // Only apply this listener when the Page model is being modified
+            if (!$widget->model instanceof \Winter\Pages\Classes\Page) {
+                return;
+            }
+
+            // Only apply this listener when the Form widget in question is a root-level
+            // Form widget (not a repeater, nestedform, etc)
+            if ($widget->isNested) {
+                return;
+            }
+	    
+	    // add fields
+            $widget->tabs['fields']['viewBag[myField]'] = [
+                'tab' => 'mytab',
+                'label' => 'myLabel',
+                'type' => 'text'
+            ];
+	    
+	    // translate fields
+            $translatable = [
+                'viewBag[myField]'
+            ];
+
+            // push the fields to the translatable array
```

Comment:
This is usually done like this:
```
$widget->model->translatable = array_merge($widget->model->translatable, $translatable);
```

Reply:
Uhh I never really worked with Github except for storing my code, can you change this or how do I do this?

Reply:
Just make the change in your local folder, then push the change to your branch and it will update the PR.

i.e.
```
[make the changes]
git commit
git push
```

Reply:
I edited the file straight on Github and the fork on my Github account is removed months ago

Reply:
You can always fork again, make the changes, push to your repo and create a new PR